storeutil/storeutil.go

```diff
@@ -11,10 +11,10 @@ import (
 	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
 )
 
-// LoaderForBlockstore returns an IPLD Loader function compatible with graphsync
-// from an IPFS blockstore
-func LoaderForBlockstore(bs bstore.Blockstore) ipld.Loader {
-	return func(lnk ipld.Link, lnkCtx ipld.LinkContext) (io.Reader, error) {
+// LinkSystemForBlockstore constructs an IPLD LinkSystem for a blockstore
```

Comment:
is this repo (graphsync) the best place for this to live? can we put it in the blockstore and/or in cidlink?
(@warpfork do you think this would be reasonable to live in cidlink?)

Reply:
That's a good question.  I don't have a strong opinion.

If we end up writing this a lot of times in a lot of places, then maybe it makes sense to put it somewhere in the go-ipld-prime repo.  Probably not the `cidlink` package directly -- because I'd like use of that package to drag in `go-cid` and nothing else -- but some other glue package of the same general pattern as `cidlink` could end up being reasonable.

Maybe putting it in the blockstore repo could make sense, too.  I think that would tend to work and should not result in cycles.

Reply:
it should migrate IMHO, eventually.